docker-compose.yml

```diff
@@ -5,17 +5,34 @@ services:
     restart: always
     volumes:
       - lodestar:/root/.local/share/lodestar
-    command: beacon --network ${NETWORK:-pyrmont}
+      - /root/.local/share/lodestar/pyrmont:/root/.local/share/lodestar/pyrmont
+    command: beacon --network ${NETWORK:-pyrmont} --api.rest.host 0.0.0.0
+    expose:
+      - "9596"
     ports:
       - "9000:9000"
 
+  validator:
+    build: .
```

Comment:
You should give the image a name, and the same in the lodestar service so it's only build once.

Reply:
i made the build happen in the beacon_node service and then used that image in validator.  if this is off in any way, i'm open to other ways